warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java

```diff
@@ -5069,6 +5069,11 @@ public static void reset(GeoTimeSerie gts) {
 
   public static List<GeoTimeSerie> map(GeoTimeSerie gts, Object mapper, long prewindow, long postwindow, long occurrences, boolean reversed, int step, boolean overrideTick, WarpScriptStack stack,
                                        List<Long> outputTicks, boolean dedup) throws WarpScriptException {
+    return map(gts, mapper, prewindow, postwindow, occurrences, reversed, step, overrideTick, stack, outputTicks, false, false);
+  }
+
+  public static List<GeoTimeSerie> map(GeoTimeSerie gts, Object mapper, long prewindow, long postwindow, long occurrences, boolean reversed, int step, boolean overrideTick, WarpScriptStack stack,
+                                       List<Long> outputTicks, boolean dedup, boolean fillOnly) throws WarpScriptException {
```

Comment:
Why add a fillOnly parameter? This value can be computed in map by checking if mapper is an instance of FILLMAPPER.